recipes/essm/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "essm" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c8d255ae82eec9430343dbe338eac31c4757511627503b5074d447c552485f20
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.0
+    - pip
+    - setuptools-scm >=3.1.0
+  run:
+    - python >=3.0
+    - sympy >=1.6.0
+
+test:
+  imports:
+    - check-manifest>=0.25
+    - coverage >=4.0
+    - isort >=5.0.0
+    - pydocstyle >=1.0.0, <4.0.0
```

Comment:
should be `>=1.0.0, <4.0.0`

Reply:
I thought there should be no version constraints?

Reply:
Which one do you mean? pydocstyle was already >=1.0.0, <4.0.0